src/portal/api/mod.rs

```diff
@@ -161,6 +167,24 @@ impl Keyring {
             .collect()
     }
 
+    pub fn lookup_item(
```

Comment:
Would add a one line doc to this function

Reply:
Does it make sense to provide a similar function to the oo7::Keyring type as well?